### PR TITLE
thuhien-week7: Develop test scenarios using features file for Opencart web application

### PR DESCRIPTION
**Pull request by thuhien**
## Exercise

* [Exercise-78](https://github.com/NT-Line-team/Behave_test/issues/9)

## Overview

Develop test scenarios using features file of below function:
Login
Register
Forgot password
Search a product
Buy a product and checkout
Add review for product

## Verification Steps

1. Open all of files .feature
2. Review scenarios for 6 features

### Docs
1. `add_review.feature`
2. `add_review.feature`
3. `forgot_password.feature`
4. `login.feature`
5. `register_opencart.feature`
6. `search_product.feature`

### CODE REVIEW

**Comment by thuhien** [comment]
Hi anh @hungphamviet ,
Please help to review homework-week7
Thanks,
Hien Truong

File: features/thuhien/add_review.feature [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Feature: Add review for product
+
```
Comment (duynguyenx) [inline features/thuhien/add_review.feature]: Missing the user role and the description about this feature. Pls see below example:
`As a standard user, I want to add a review for product that i bought`

Pls also correct all features file below

File: features/thuhien/add_review.feature [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  @id-test: 001
```
Comment (duynguyenx) [inline features/thuhien/add_review.feature]: It should be `@id-001`
Pls also correct all features file below

**Comment by thuhien** [comment]
Hi @duynguyenx ,
Please help to review again.
Thanks,
Hien Truong

File: features/thuhien/add_review.feature [hunk 3]
```diff
@@ -0,0 +1,96 @@
+Feature: Add review for product
+  As a standard user
+  I want to add review for product
+  so that there is more information for another people
+
+  @id-test: 001
```
Comment (duynguyenx) [inline features/thuhien/add_review.feature]: Should not use colon when put tag for test case. I suggest to use `id-test-001`. Pls also correct other tags

**Comment by thuhien** [comment]
Hi @duynguyenx ,
Duy helps to re-check it.
Thanks,
Hien Truong

**Comment by duynguyenx** [comment]
LGTM

**Pull request merged**